modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1334 @@
+//#TINY-8872
+
+In previous versions of {productname}, It was discovered that the editor had some issues with the autoresize function when using content_css documents, which caused the editor to have a indefinite height when content was being added.
+
+To fix this issue, {productname} 6.4.1 fixed this issue by addressing the conflict between the document skin's CSS and the autoresize bottom margin setting. After this fix, the resize function now checks for loops and considers the margin bottom as always set to 0 instead of autoresize bottom margin, which ensutes that the editor's height is calculated correctly. 
+
+=== Plugin - Search and Replace - Throws a JS console error if you search for a string that doesn't exist and try to click on the preferences button.
+//#TINY-9686
+
+In previous versions of {productname}, the 'Search and Replace' plugin had a known issue that caused a console error when searching for a string that wasn't in the editor content, and then attempting to click on the `preferences` button.
+
+As a result, an "incorrect type" error would prevent users from opening the preferences section.
+
+In {productname} 6.4.1, this issue has been resolved.
+
+=== Toolbar doesn't hide when editor is out of view in scrollable container
+//#TINY-9652
+
+In previous versions of {productname}, when the editor is within a scrollable container, the toolbar would remain visible when the editor is scrolled out of view.
+
+With the release of {productname} 6.4.1, the new `ui_mode` option provides improvements to editors within scrollable contexts, including hiding the toolbar when the editor is out of view when the `ui_mode` option is set to 'split'.
+
+=== `color_cols`: 0 Memory Leak
+//#TINY-9649
+
+In previous versions of {productname}, the color swatch would attempt to display 0 columns of colors when the relevant `color_cols`, `color_cols_foreground` or `color_cols_background` options were set to 0. As a consequence, the tab would crash when opening that color swatch.
+
+In  {productname} 6.4.1, the default number of columns, 5, are now used when any of these options is set to 0 or less, preventing this issue from occurring.
+
+
+[[known-issues]]
+== Known issues
+
+This section describes issues that users of {productname} 6.4.1 may encounter and possible workarounds for these issues.
+
+There are several known issues in {productname} 6.4.1.
+
+=== Accessibility check dropdown not populated when switching between issues
+//#TINY-9679
+
+In previous versions of {productname}, a bug was discovered that encountered an issue with the **Accessibility Checker** dropdown. When switching between different issues, the dropdown would not re-populate on existing viewed issues.
+
+As a result, improvements were made to the **Accessibility Checker** dropdown with {productname} 6.4.1, to ensure that users can now switch between issues with confidence that the dropdown will populate as expected.
```

Comment:
```suggestion
In previous versions of {productname}, a bug was discovered that encountered an issue with the **Accessibility Checker** dropdown. When switching between different issues, the dropdown would not re-populate on existing viewed issues.
```

Reply:
@MitchCTiny moved this into this review, as it was outdated.
Reference comment: This isn't merged yet, and won't be in 6.4, so it shouldn't be documented as complete. It's in the known issues section.